src/lfortran/semantics/ast_common_visitor.h

```diff
@@ -3094,15 +3094,20 @@ class CommonVisitor : public AST::BaseVisitor<Derived> {
                 if( m_value && m_value->type == ASR::exprType::StringConstant ) {
                     ASR::StringConstant_t *m_str = ASR::down_cast<ASR::StringConstant_t>(m_value);
                     std::string sliced_str;
+                    int str_length = 0;
+                    while( m_str->m_s[str_length] != '\0' ) {
+                        str_length += 1;
+                    }
```

Comment:
Can we simply use `strlen()` here?
```suggestion
                    int str_length = strlen(m_str->m_s);
```

Reply:
Done 👍🏻